SublimeText/Preferences.md

```diff
@@ -2,7 +2,7 @@
 
 This is an example of User Settings for a basic development but please feel free to modify or update as per your choice.
 
-~~~json
+```json
```

Comment:
Did you verify how this looks? `make serve` to bring it up locally in case you didn't know